src/SingleBookingPage/bookingItem/AccordionHeader.js

```diff
@@ -63,7 +63,7 @@ const AccordionHeader = ({ trip, isToggled }: Props) => {
         <div className="itinerary">
           <span className="location">
             <Text variant="bold" type="primary">
-              {departureCity}&nbsp;({departureCode})
+              {departureCity}&nbsp;{departureCode}
```

Comment:
The lack of parenthesis is correct for the AllBookings page, however the singleBooking page has parenthesis around the city code according to design  doesn't it?
https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/screen/5aaac0e6c41e3efb0dcd168a

Reply:
As I see in this https://github.com/kiwicom/smart-faq/issues/99#issuecomment-393114899 ```Abbreviations should not be in parenthesis. It has been a decision made by designers here in our team, mainly to make it look less technical.```

this desing for example dosen't have parenthesis https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/screen/5adf5b9a2b04e703bf034c36

Reply:
You're right, didn't see that comment.